api/client/webclient/webconfig.go

```diff
@@ -106,6 +106,8 @@ type WebConfig struct {
 	SAML bool `json:"saml"`
 	// MobileDeviceManagement indicates whether adding Jamf plugin is enabled
 	MobileDeviceManagement bool `json:"mobileDeviceManagement"`
+	// IsCommunity determines if this binary is running the community edition
+	IsCommunity bool `json:"isCommunity"`
```

Comment:
Could we deprecate `IsEnterprise` and instead have a string `Edition` which can be `agpl`, `community`, or `enterprise`?

Reply:
I opted to use the `BuildType()` here, so it does the same thing, but the values are `oss`, `ent`, or `community` now.